sunpy/__init__.py

```diff
@@ -22,7 +22,6 @@
 try:
     from .version import __version__
 except ImportError:
-    print("version.py not found, please reinstall sunpy.")
```

Comment:
We should keep this or something like it.

Reply:
Undone.